### PR TITLE
Hn ah make ticket post to ticket

### PR DESCRIPTION
**Pull request by a-holliday**

### CODE REVIEW

File: BugTracker/src/main/java/BugTracker/ServerDriver.java [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
+import BugTracker.pojos.Status;
+import BugTracker.pojos.Ticket;
+import BugTracker.pojos.UserRole;
+
 @Configuration
 @ComponentScan
 public class ServerDriver {
+	@Autowired
+	SessionFactory sessionFactory;
+	TicketDao ticketDao = new TicketDaoImpl(sessionFactory);
```
Comment (JoeDKoal94) [inline BugTracker/src/main/java/BugTracker/ServerDriver.java:28]: >Remove this line, no purpose being

File: BugTracker/src/main/java/BugTracker/ServerDriver.java [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
 
 import org.apache.catalina.LifecycleException;
 import org.apache.catalina.startup.Tomcat;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
+import BugTracker.dao.TicketDao;
+import BugTracker.dao.TicketDaoImpl;
```
Comment (JoeDKoal94) [inline BugTracker/src/main/java/BugTracker/ServerDriver.java:16]: Remove uneeded imports

File: BugTracker/src/main/java/BugTracker/dtos/TicketDto.java [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+		this.ticketId = ticketId;
+		this.openerId = openerId;
+		this.assignedDeveloperId = assignedDeveloperId;
+		this.name = name;
+		this.created = created;
+		this.statusId = statusId;
+		this.priorityId = priorityId;
+		this.difficultyLevel = difficultyLevel;
+		this.commentIds = commentIds;
+	}
```
Comment (JoeDKoal94) [inline BugTracker/src/main/java/BugTracker/dtos/TicketDto.java:57]: > Probably consider removing ticketId, assignedDeveloperId, statusId, thesewill be automatically generated in the java

**Comment by kmilstein** [comment]
Talked about notes verbally

**Review by kmilstein — COMMENTED** [review]
Talked about comments verbally on Zoom

**Review by JoeDKoal94 — APPROVED** [review]

**Pull request merged**